app/main/views/users.py

```diff
@@ -295,10 +294,6 @@ def email_has_valid_buyer_domain():
         abort(400, "'email_address' is a required parameter")
 
     domain_ok = buyer_email_domain_approved(BuyerEmailDomain.query.all(), email_address.split('@')[-1])
-    if not domain_ok:
-        # If the data migration hasn't happened yet, check the .txt file
```

Comment:
The migration is definitely there! 😸  https://github.com/alphagov/digitalmarketplace-api/blob/master/migrations/versions/1030_buyer_email_domain_data_migration.py

Reply:
Oh thanks, the ordering of numbers alphabetically meant I didn't see it. Although just to confirm, if any developer or new joiner ran the migration after this PR they would not end up with the domains in their DB because we are about to delete the file?

Reply:
Yes, I think so. We'd have to do a fresh DB dump 😸